src/internal_rep.rs

```diff
@@ -248,6 +248,24 @@ impl Declared for TypeInfo {
 
 impl TypeInfo {
     pub fn new(td: TypeDecl, file: &SimpleFile<String, String>) -> Result<TypeInfo, CascadeErrors> {
+        let mut temp_vec = td.inherits.to_vec();
+        temp_vec.sort();
+        let mut iter = temp_vec.iter().peekable();
+        while let Some(cur_val) = iter.next() {
+            match iter.peek() {
+                Some(next_val) => {
+                    if cur_val == *next_val {
+                        return Err(CascadeErrors::from(ErrorItem::Compile(CompileError::new(
+                            "Duplicate Inherit",
+                            file,
+                            (*next_val).get_range(),
+                            "Duplicate Inherit. Possible Typo?",
```

Comment:
The additional help here shouldn't restate what the first message said directly.  I'd prefer something more verbose, like "This type to inherit is identical to another type in the same inheritance list.  Perhaps you meant to inherit some other type?"